### PR TITLE
Reverts looping delta alert noise

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
Partially reverts #21174, but leaving the framework for decreasing sounds.

## Why It's Good For The Game
This sound really hurts my ears after a while, and I have heard from others they feel the same. Yes this option exists in the mixer, but:
- It doesnt work because the sounds volume override trumps it (I think thats the reason atleast)
- Few people actually use the mixer since its not super obvious, which leads to everyone hearing this
    - It would be much more obvious as a "hear/silence delta alarm", but we already have enough of this

## Testing
Primed nuke, ears didnt implode

## Changelog
:cl:
del: Removes looping siren when nuke is primed
/:cl:

---

I do not class this a "reactionary revert", as its been a good 3 weeks since the original PR was merged. Headcoders may dispute this.

### CODE REVIEW

**Comment by DGamerL** [comment]
Good stuff, the Delta sound is very loud. The idea behind it is good, but it needs some tweaking before it's added again

File: code/modules/client/preference/preferences.dm [hunk 1]
```diff
@@ -88,7 +88,6 @@ GLOBAL_LIST_INIT(special_role_times, list( //minimum age (in days) for accounts
 		"1017" = 100, // CHANNEL_ENGINE
 		"1016" = 100, // CHANNEL_FIREALARM
 		"1015" = 100, // CHANNEL_ASH_STORM
-		"1014" = 100, // CHANNEL_DELTA_ALARM
```
Comment (Contrabang) [inline code/modules/client/preference/preferences.dm:91]: Will this have any problems with the database? If someone set the channel 1014 to 5, and then in the future the game selects 1014 as a random channel, would the sound be randomly very quiet?

**Comment by 1080pCat** [comment]
Honestly, im 50/50 for this.. Maybe making it a preference to change it might be the best middle ground?

Semi related to my suggestion is if it is made a preference, admin midis should stop the sound all together, to let people hear the,, music admins play at the end of nukes without hearing the alarm blaring.

File: code/modules/client/preference/preferences.dm (hunk 1 above)
Comment (AffectedArc07) [reply code/modules/client/preference/preferences.dm:91]: I can run a script to wipe out the 1014 entry.

**Comment by S34NW** [comment]
A better alternative might be to make the sound local to the nuke instead of direct play to all z mobs

**Comment by tipoima** [comment]
> A better alternative might be to make the sound local to the nuke instead of direct play to all z mobs

The idea was that it's a station-wide alarm though. Making it local makes it redundant (and I guess makes it easier to find in extremely rare cases?)

**Comment by CinnamonSnowball** [comment]
All alert sound effects are somewhat loud, the main difference is that they're short.
I think the proper solution would be to just make the looping alert soundclip fade out stronger and faster (hyperbolically rather than linearly) to bring it closer to the other alerts, and letting it stay it a long-term quiet sound that adds a sense of urgency while there's an *active threat of nuking the whole station, including you.*

**Review by S34NW — APPROVED** [review]

**Pull request merged**